components/shell/ConfigurationContext.tsx

```diff
@@ -164,3 +164,14 @@ export function useServerConfig() {
 export function usePermission(name: PermissionName) {
   return useServerConfig().permissions[name]
 }
+
+export function useAppviewAgent() {
+  const { appview } = useConfigurationContext().serverConfig
+
+  return useMemo<Agent | null>(() => {
+    if (appview) {
+      return new Agent(new CredentialSession(new URL(appview)))
```

Comment:
Doesn't this work ?

```suggestion
      return new Agent(appview)
```

Reply:
unfortunately, no. tried to explain that here https://github.com/bluesky-social/ozone/pull/162#discussion_r1801822826

<img width="1109" alt="Screenshot 2024-10-16 at 20 27 50" src="https://github.com/user-attachments/assets/067196ab-c1a4-4e8d-8515-ff3dc2098805">